package.json

```
@@ -13,7 +13,7 @@
     "hooks:pre-commit": "node hooks/pre-commit.js",
     "commit": "git-cz",
     "release:first": "yarn release -- --first-release",
-    "build:lib": "ng build ngworker-lumberjack --prod && npx copy README.md ./dist/ngworker/lumberjack",
+    "build:lib": "ng build ngworker-lumberjack --prod",
```

We want to add `README.md` to the bundle.
```suggestion
    "build:lib": "ng build ngworker-lumberjack --prod && npx copy README.md ./dist/ngworker/lumberjack",
```